hw2/03-count.js

```diff
@@ -1,3 +1,33 @@
 // Add your code here
 
-input.addEventListener('keydown', handleKeyDown);
+const inputElement = document.getElementById('userInput');
+const textElement = document.getElementById('textContainer');
+
+const highlightMatchingWords = (event) => {
+  // Execute the function only when the Enter key is pressed
+  if (event.key !== 'Enter') {
+    return;
+  }
+
+  const userInput = inputElement.value.trim().toLowerCase();
+  if (!userInput) {
+    return; // Do nothing if input is empty
+  }
+
+  const text = textElement.textContent;
+  const words = text.split(/\b/); // Split the text into words and punctuation
+  const highlightedText = words.map((word) => {
+    // Check if the word matches the user input, ignoring case
+    if (word.toLowerCase() === userInput) {
+      return `<span class="highlight">${word}</span>`; // Highlight matching words
```

Comment:
<mark> can be used instead of creating a class but however this approach is useful if the user ever wants to change the color of the highlight.